Makefile

```diff
@@ -131,6 +131,12 @@ test: manifests generate fmt vet envtest ## Run tests. We need LOCALBIN=$(LOCALB
 	mkdir -p $(LOCALBIN)/default-config && cp config/manager/$(CONF_DIR)/* $(LOCALBIN)/default-config
 	LOCALBIN=$(LOCALBIN) KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test $(PKGS) -coverprofile cover.out
 
+.PHONY: integration-test
+integration-test: ginkgo install fmt vet envtest ## Run integration_tests. We need LOCALBIN=$(LOCALBIN) to get correct default-config path
```

Comment:
```suggestion
integration-test: ginkgo manifests generate fmt vet envtest ## Run integration_tests. We need LOCALBIN=$(LOCALBIN) to get correct default-config path
```

As discussed already, otherwise this would expect a running cluster:
```
$ KUBECONFIG=/dev/null make integration-test
test -s /home/asoro/work/projects/backstage/janus-idp/operator/bin/ginkgo || GOBIN=/home/asoro/work/projects/backstage/janus-idp/operator/bin go install github.com/onsi/ginkgo/v2/ginkgo@v2.17.1
test -s /home/asoro/work/projects/backstage/janus-idp/operator/bin/controller-gen || GOBIN=/home/asoro/work/projects/backstage/janus-idp/operator/bin go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.11.3
/home/asoro/work/projects/backstage/janus-idp/operator/bin/controller-gen rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
test -s /home/asoro/work/projects/backstage/janus-idp/operator/bin/kustomize || { curl -Ss "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh" | bash -s -- 3.8.7 /home/asoro/work/projects/backstage/janus-idp/operator/bin; }
/home/asoro/work/projects/backstage/janus-idp/operator/bin/kustomize build config/crd | kubectl apply -f -
Error from server (NotFound): the server could not find the requested resource
make: *** [Makefile:193: install] Error 1
```